Sources/DDGSync/SyncFeatureEntity.swift

```diff
@@ -31,8 +31,9 @@ public class SyncFeatureEntity: NSManagedObject {
     }
 
     @NSManaged public var name: String
-    @NSManaged public internal(set) var lastModified: String?
     @NSManaged private var state: String
+    @NSManaged public internal(set) var lastModified: String?
+    @NSManaged public internal(set) var lastSynced: Date?
```

Comment:
`lastSynced` represents the local timestamp. The name is to be changed because it's super confusing in comparison to `lastModified`. On a positive side, it's not used anywhere outside of `SyncMetadataStore`.